bower.json

```diff
@@ -27,14 +27,36 @@
     "package.json"
   ],
   "dependencies": {
-    "d2l-colors": "^2.3.0",
-    "d2l-offscreen": "^2.2.4",
-    "iron-iconset-svg": "polymerelements/iron-iconset-svg#^1.1.1",
-    "polymer": "^1.9.0"
+    "d2l-colors": "2.3 - 3.0",
+    "d2l-offscreen": "2.2 - 3.0",
+    "iron-iconset-svg": "polymerelements/iron-iconset-svg#1 - 2",
+    "polymer": "Polymer/polymer#1.9 - 2"
   },
   "devDependencies": {
-    "d2l-demo-template": "^0.0.12",
-    "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.4.1",
-    "web-component-tester": "^5.0.0"
+    "d2l-demo-template": "0.0.12 - 1.0",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#1 - 2",
+    "web-component-tester": "^6.0.0"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "d2l-colors": "^2.3.0",
+        "d2l-offscreen": "^2.2.4",
+        "iron-iconset-svg": "polymerelements/iron-iconset-svg#^1.0.0",
+        "polymer": "Polymer/polymer#^1.9.1"
+      },
+      "devDependencies": {
+        "d2l-demo-template": "^0.0.12",
+        "iron-test-helpers": "PolymerElements/iron-test-helpers#1.4.1",
+        "web-component-tester": "^5.0.0"
+      },
+      "resolutions": {
+        "webcomponentsjs": "^0.7",
+        "polymer": "Polymer/polymer#^1.9.1"
+      }
+    }
+  },
+  "resolutions": {
+    "webcomponentsjs": "0.7 - 1.0.0"
```

Comment:
Will all these ranges that don't have `^`s work with future minor/bug fix releases?